exercises/clock/src/test/kotlin/ClockEqualTest.kt

```diff
@@ -0,0 +1,101 @@
+import org.junit.Ignore
+import org.junit.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+/*
+ * version: 1.0.1
+ */
+class ClockEqualTest {
+
+    @Ignore
+    @Test
+    fun clocksWithSameTimeAreEqual() {
+        assertTrue(Clock(15, 37) == Clock(15, 37))
```

Comment:
Would it be more clear to just use `assertEquals` and `assertNotEquals` for these tests?